pkg/kubectl-argo-rollouts/info/rollout_info.go

```diff
@@ -326,3 +329,35 @@ func (r *RolloutInfo) AnalysisRunsByRevision(rev int) []AnalysisRunInfo {
 	}
 	return runs
 }
+
+func addSteps(stepsIndex int, index *int32, stepsData *[]string, steps v1alpha1.CanaryStep, ro *v1alpha1.Rollout) {
+	currentIcon := ""
+	if int32(stepsIndex)+1 == *index && *index <= int32(len(ro.Spec.Strategy.Canary.Steps)) {
+		currentIcon = "[*]"
+	}
+	if steps.SetWeight != nil {
+		*stepsData = append(*stepsData, fmt.Sprintf("%vsetWeight:%v", currentIcon, *steps.SetWeight))
+	}
+	if steps.Pause != nil {
+		pause := fmt.Sprintf("%vpause:always", currentIcon)
+		if steps.Pause.Duration != nil {
+			pause = fmt.Sprintf("%vpause:%vs", currentIcon, steps.Pause.Duration)
+		}
+		*stepsData = append(*stepsData, pause)
+	}
```

Comment:
We should also be printing steps for things other than `pause` and `setWeight`. For example: analysis, experiment, and setCanaryScale steps.

Reply:
I'm in an environment where I can't run these, so I can't see the results。
If you can, you can modify it on this basis.